LRAnalysisLauncher/Program.cs

```diff
@@ -59,7 +59,7 @@ static int Main(string[] args)
             int iPassed = (int)Launcher.ExitCodeEnum.Passed;//variable to keep track of whether all of the SLAs passed
             try
             {
-                if (args.Length != 3)
+                if (args.Length != 3 && args.Length != 4)
```

Comment:
Whats wrong with length 3 or 4, It missing some explanation or move it to a function with describing name?